x-pack/plugins/security_solution/public/detection_engine/rule_creation/components/threat_match_index_edit/validators/forbidden_index_pattern_validator.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ValidationFunc } from '../../../../../shared_imports';
+import { customValidators } from '../../../../../common/components/threat_match/helpers';
+
+export const forbiddenIndexPatternValidator: ValidationFunc = (...args) => {
+  const [{ value }] = args;
+
+  return customValidators.forbiddenField(value, '*');
```

Comment:
This bug is reproducible in `main` as well. I noticed it earlier and intended to create a bug ticket. But I changed my mind to first check if there is a trivial fix. In that case it could be fixed in this PR.